packages/shorebird_cli/lib/src/executables/powershell.dart

```diff
@@ -47,6 +47,10 @@ class Powershell {
       runInShell: true,
     );
 
-    return (result.stdout as String).trim();
+    var versionString = (result.stdout as String).trim();
+    if (!versionString.contains('+')) {
+      versionString += '+0';
```

Comment:
I wonder if this belongs here or somewhere deeper (to cover other platforms with the same bug)?

Reply:
As of now, Windows is the only platform with this issue, primarily because we determine the release version differently in the CLI than we do from the executable. Leaving here for now.